lib/ansible/plugins/doc_fragments/action_common_attributes.py

```diff
@@ -14,13 +13,6 @@ class ModuleDocFragment(object):
     action:
       description: Indicates this has a corresponding action plugin so some parts of the options can be executed on the controller
       support: none
-    action_group:
-      description: Action is part of action_group(s), for convenient setting of module_defaults.
-      support: none
-      membership: []
```

Comment:
Why remove `action_group`? I think this is a very useful attribute.

Reply:
Oh, good spotting

Reply:
was part of the reduction of 'nothing in core uses it' ... its mostly used by cloud and networking modules, happy to put it back if the majority think it is needed here.

Reply:
If documenting core features not utilized by core modules is moved for things like turbo/action_groups, it would be nice if there was general documenttion somewhere (or maybe a validator) of standardized attribute names so collections document things consistently.

Reply:
i had thought of that but gave up since we couldn't even name options consistently with everything in the same repo, much less after the collection diaspora.

Reply:
I think the general idea (at least how I see it) is that there are doc fragments (quasi-)standardizing the attribute names, and  assuming attribute users (i.e. modules) don't supply required arguments such as `description`, ansible-test does some validation by making sure that you only use attribute names that come from docs fragments you include.

If the turbo fragment lives next to the turbo code, and everyone using the turbo code uses that docs fragment, the name (and description) of the attribute(s) in there is basically standardized.

Reply:
@felixfontein That's how I see the doc fragments too. I liked them in core because it standardizes it across collections. I'm not particular about the turbo case (didn't realize it was at no time a part of ansible/ansible), it just seemed similar to action_groups in that multiple collections could implement it without a common dependency (other than ansible-core). Anyway, since action_groups was put back by "suggestion" and if the comment doesn't apply to turbo then disregard it.